packages/ndla-ui/src/Aside/Aside.tsx

```diff
@@ -6,42 +6,107 @@
  *
  */
 
-import { ReactNode } from 'react';
-import BEMHelper from 'react-bem-helper';
-
-const classes = BEMHelper({
-  prefix: 'c-',
-  name: 'aside',
-});
-
-interface Props {
-  dangerouslySetInnerHTML?: {
-    __html: string;
-  };
-  children?: ReactNode;
+import { ComponentProps, useMemo } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { breakpoints, colors, fonts, mq, spacing } from '@ndla/core';
+
+interface Props extends ComponentProps<'aside'> {
   narrowScreen?: boolean;
   wideScreen?: boolean;
   alwaysShow?: boolean;
 }
 
-const Aside = ({
-  children,
-  narrowScreen = false,
-  dangerouslySetInnerHTML,
-  wideScreen = false,
-  alwaysShow = false,
-}: Props) => {
-  const modifiers = {
-    narrowScreen,
-    wideScreen,
-    alwaysShow,
-  };
+const StyledAside = styled.aside`
+  position: relative;
+  margin: ${spacing.large} 0px;
+  ${mq.range({ from: breakpoints.tablet })} {
+    max-width: 350px;
+    float: right;
+    clear: right;
+    width: 50%;
+    left: auto !important;
+    padding: 0;
+    padding-left: ${spacing.small};
+    ${mq.range({ from: breakpoints.desktop })} {
+      width: 75%;
+    }
+  }
+
+  border-left: 4px solid ${colors.background.dark};
+`;
+
+const wideScreenStyle = css`
+  display: none;
+  ${mq.range({ from: breakpoints.tablet })} {
+    display: block;
+  }
+`;
+
+const narrowScreenStyle = css`
+  ${mq.range({ from: breakpoints.tablet })} {
+    display: none;
+  }
+`;
+
+const alwaysShowStyle = css`
+  display: block !important;
+`;
+
+const StyledAsideContent = styled.div`
+  color: ${colors.brand.greyDark};
+  padding: ${spacing.normal};
+
+  & > *:first-child {
+    margin-top: 0;
+  }
+  & > *:last-child {
+    margin-bottom: 0;
+  }
+
```

Comment:
Var det meining å droppe img, figure og h1?

Reply:
Ja, men med forskjellige grunner:
* *h1*: Vi har ikke h1 i artikler. 
*  *img*: Vi bruker ikke en box-shadow for bilder noe annet sted. Hvorfor skal vi gjøre det i kun aside?
* *figure*: Dette henger nok igjen fra tiden der hver `figure` var større enn siden. Nå er bredden til figures satt til 100%.